.circleci/config.yml

```diff
@@ -95,7 +95,8 @@ commands:
           no_output_timeout: 16m
           command: |
             mkdir -p ${SCREENSHOT_DIR}
-            yarn test --ci --maxWorkers=2 --reporters=default --reporters=jest-junit
+            TESTS_TO_RUN=$(circleci tests glob "jest/**.integration-test.js" | circleci tests split --split-by=timings)
+            yarn test $TESTS_TO_RUN --ci --maxWorkers=2 --reporters=default --reporters=jest-junit
```

Comment:
Do we still want `--maxWorkers=2`?

Reply:
Yes. Each container has 2 cpu threads available, and will use them if the container is assigned more than one test.